### PR TITLE
Added Rendezvous Related Utility Functions

### PR DESCRIPTION
**Pull request by kylekrol**
The functions were tested with the `MATLAB/rendezvous/rendezvous.m` script.

### CODE REVIEW

File: MATLAB/utl/utl_orbrate.m [hunk 1]
```diff
@@ -0,0 +1,8 @@
+function n = utl_orbrate(a)
+% Calculates the orbital rate of the sattelite assuming a circular orbit
```
Comment (nhz2) [inline MATLAB/utl/utl_orbrate.m]: Add what units a and n are to the description.

**Review by nhz2 — CHANGES_REQUESTED** [review]
Looks good, just one issue with units in the docs. Also maybe some of these functions should be in the rendezvous folder, or a helper function sub folder, if they are only used for rendezvous, like the l1cost functions.

File: MATLAB/utl/utl_orbrate.m (hunk 1 above)
Comment (kylekrol) [reply MATLAB/utl/utl_orbrate.m]: Yeah good point - MKS for everything.

**Comment by kylekrol** [comment]
I'm gonna migrate some of the rendezvous specific code to `MATLAB/rendezvous` as suggested and will fix that comment too.

**Review by nhz2 — APPROVED** [review]

**Pull request merged**